### PR TITLE
552 nav bar mobile

### PR DESCRIPTION
**Pull request by JpadillaCoding**
Issue #552 

- Left aligned the Home Unite Us logo. 
    - This meant replacing the hamburger menu
    - Kept the functionality of the hamburger menu.
    - Referred to the Figma file on the issue
- Right avatar icon was already aligned
    - Noticed a bug on mobile that when the icon is clicked, the nav bar is shifted to the left. Added ``overflowY: 'scroll'`` to the navbar to keep this from happening. This seems to be a material UI [issue](https://github.com/mui/material-ui/issues/10000)
    - The logout button now takes the user to the homepage. Made this change on the ``header.tsx`` file, but was being overridden by the ``ProtectedRoute.tsx`` file. Made the change there also.
   

<details>
  <summary>Preview Before</summary>

  https://github.com/hackforla/HomeUniteUs/assets/104275658/6942c130-a249-48fc-8328-45239cbb008a

Note: The right item shifting
</details>
<details>
  <summary>Preview With Changes</summary>

  https://github.com/hackforla/HomeUniteUs/assets/104275658/48301295-e619-49df-b0fe-bccc3bd85182

</details>

### CODE REVIEW

**Comment by Joshua-Douglas** [comment]
Hey @JpadillaCoding,

Thanks for the submission! 

Did you try out these changes on a real phone? I tried these changes on my iPhone 12 using the DuckDuckGo & Chrome browser, but I wasn't able to select the login or signup buttons. I had the same issue using the `/signin` page as well. 

I haven't had time to look at the code changes yet, but I can help troubleshoot if you need it. You should be able to load the app on your phone if it is connected to the same wifi network and you update the `vite.config.ts` `defineConfig` `server` field to list `host: `0.0.0.0`.

https://github.com/hackforla/HomeUniteUs/assets/22138019/b95964ef-50fc-40cc-ab36-81611fd57a6d

Also - is the `MUI` label w/in the side panel component default component label, or does that have a valid meaning? I'm wondering if we should remove it.

**Comment by Joshua-Douglas** [comment]
Hey again @JpadillaCoding,

Apologies for the confusion, but a real phone isn't necessary. These buttons are also broken on desktop if you decrease the width of the browser enough. 

With these broken buttons mobile users can only sign in if they know to type in the `/signin` endpoint. Most users probably won't know to do this. Is it possible to fix this on this PR?

Thanks again!

**Comment by Joshua-Douglas** [comment]
I took another look using my mobile phone again. I tried creating a new account but I wasn't able to use the 'Sign up' or 'Continue with Google' button. 

I was able to create and login to accounts without issue on desktop. If I try to sign into those accounts on mobile, however, I run into the same issue. I can't press the 'Sign In' button. 

I wonder if anyone else is having the same issue.

**Comment by paulespinosa** [comment]
@Joshua-Douglas Firefox and Chrome have responsive design mode and profiles for different mobile/tablet devices that can be used to test mobile.

The "Login" and "Sign up" buttons don't work for me either. In mobile mode, the source code shows there aren't `href`s on those buttons, but in desktop mode there are `href`s.

**Comment by Joshua-Douglas** [comment]
> @Joshua-Douglas Firefox and Chrome have responsive design mode and profiles for different mobile/tablet devices that can be used to test mobile.
> 
> 
> 
> The "Login" and "Sign up" buttons don't work for me either. In mobile mode, the source code shows there aren't `href`s on those buttons, but in desktop mode there are `href`s. But maybe that's irrelevant. 

Appreciate the tip! I'll check those out.

**Comment by brycelednar** [comment]
Hi @JpadillaCoding and team, 

Thanks for all of the work on this @JpadillaCoding !

To clarify, the navigation bar updates described in #552 are relevant for users who have already logged in (no updates to the existing pre-login landing page, dev.homeunite.us, as part of this issue). Bugs on the pre-login landing page are out of scope for #552. I'll file bugs to address those separately.

Regarding feedback on the updates shown in the "Preview With Changes" screen capture above, there should not be a dropdown from the left-aligned Home Unite Us logo (the dropdown functionality that you included from the avatar is correct). 

Let me know of any further questions or clarifications. Thanks again for all of the great work on this!

Bryce

**Comment by JpadillaCoding** [comment]
@brycelednar I will remove the MUI tab on the logo. Can you clarify If this tab will be used at all in the future? There is other code wrapped around it and I would like to know if I should clean it up depending on if we will use it or not.

**Comment by JpadillaCoding** [comment]
@Joshua-Douglas To clarify, are you referring to the login/signup not working on the MUI tab?

**Comment by brycelednar** [comment]
> @brycelednar I will remove the MUI dropdown tab on the logo. Can you clarify If this tab will be used at all in the future? This is the only place that code is used and I would like to know if I should clean it up depending on if we will use it or not.

HI @JpadillaCoding, the plan to remove the MUI dropdown tab from the logo sounds good (again, this change is limited to the post login landing page, no changes are needed to the pre-login landing page). There are no immediate plans for a dropdown from the logo on the post-login landing page. Thanks!

**Comment by JpadillaCoding** [comment]
@brycelednar understood. In that case, the changes are ready for review.

**Review by erikguntner — APPROVED** [review]
Approving this PR. I left some comments about potential changes, but since there's going to be updates to the header in the future these could be addressed then.

File: app/src/components/common/AppLayout.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ export const AppLayout = () => {
         sx={{
           minHeight: '100vh',
           width: '100vw',
+          overflowY: 'scroll',
```
[the remark this answers is not in the document]
Comment (mira-kine) [reply app/src/components/common/AppLayout.tsx:15]: @JpadillaCoding @erikguntner I looked more into the [issue](https://github.com/mui/material-ui/issues/10000), seems like you can work directly on `Menu` and add a disableScrollLock = true? I haven't been able to try it out directly but the [Menu API](https://mui.com/material-ui/api/menu/) has a prop class `Popover Classes` which seems to have "disableScrollLock". I will try it out myself later as well, but could be better than adding overflowY to the main tag.

**Review by mira-kine — APPROVED** [review]
Added comment about overflowY on main tag. Otherwise approved.

**Comment by JpadillaCoding** [comment]
@mira-kine @erikguntner Good catch on the overflow fix being an issue. I tested out your recommendation Mira and it does stop the items from shifting, but it unveiled a separate bug on this specific page that is causing the right nav bar item to shift completely to the right. I am looking into a fix for this page-specific bug.

**Comment by JpadillaCoding** [comment]
@erikguntner @mira-kine The issue seems to be localized to the styling of `CoordinatorFashboard.tsx` with the `Grid` layout. Adding `overflowY: auto` fixes this bug. 
The issue is that this is adding a fix to code that isn't the final design. I did address the accessibility issue since that can be easy to miss in future changes.

**Comment by Joshua-Douglas** [comment]
Hey @mira-kine and @erikguntner, 

I see you both approved this issue. Do you think we are good to merge?

**Comment by erikguntner** [comment]
@JpadillaCoding looks like there is a conflict on the main branch. Can you resolve this? Then it's good to merge!

**Pull request merged**